src/plugins/legacy/itkProcessRegistrationOptimus/itkProcessRegistrationOptimusToolBox.cpp

```diff
@@ -0,0 +1,257 @@
+    d->samplesBox->setMaximum(1000000);
+    d->samplesBox->setSingleStep(1);
+    d->samplesBox->setValue(10000);
+    d->samplesBox->setToolTip(tr("Number of spatial samples"));
+
+    //Not used now, we use the default
+//    d->interpolatorTypeBox = new QComboBox(widget);
+//    d->interpolatorTypeBox->addItem(tr("nearest neighbor"),"nearest neighbor");
+//    d->interpolatorTypeBox->addItem(tr("linear"),"linear");
+//    d->interpolatorTypeBox->addItem(tr("b-spline"),"b-spline");
+//    d->interpolatorTypeBox->addItem(tr("sinc"),"sinc");
+
+    d->interpolationsBox = new QSpinBox(widget);
+    d->interpolationsBox->setMinimum(0);
+    d->interpolationsBox->setMaximum(100000);
+    d->interpolationsBox->setSingleStep(1);
+    d->interpolationsBox->setValue(28);
+    d->interpolationsBox->setToolTip(
+                tr("Set the number of interpolation points"));
+
+    d->rhoStartBox = new QDoubleSpinBox(widget);
+    d->rhoStartBox->setMinimum(0.0);
+    d->rhoStartBox->setSingleStep(0.1);
+    d->rhoStartBox->setValue(0.6);
+    d->rhoStartBox->setToolTip(
+                tr("Starting value of rho (See reference on the algorithm "
+                   "for more information)"));
+
+    d->rhoEndBox = new QDoubleSpinBox(widget);
+    d->rhoEndBox->setMinimum(0.0);
+    d->rhoEndBox->setSingleStep(0.01);
+    d->rhoEndBox->setDecimals(4);
+    d->rhoEndBox->setValue(0.003);
+    d->rhoEndBox->setToolTip(
+                tr("Stopping value of rho (See reference on the algorithm "
+                   "for more information)"));
+
+    d->scalingCoeffBox = new QDoubleSpinBox(widget);
+    d->scalingCoeffBox->setMinimum(0.0);
+    d->scalingCoeffBox->setSingleStep(0.1);
+    d->scalingCoeffBox->setValue(50.0);
+    d->scalingCoeffBox->setToolTip(tr("Scaling coefficient"));
+
+    this->setTitle(tr("Optimus"));
+    layout->addRow(new QLabel(tr("Max. number of iterations"),widget),
+                   d->iterationsBox);
+
+    layout->addRow(new QLabel(tr("Number of Histogram bins"),widget),d->binsBox);
+    layout->addRow(new QLabel(tr("Number of samples"),widget),d->samplesBox);
+    //Not used now
+//    layout->addRow(new QLabel(tr("Interpolation Type"),widget),
+//                   d->interpolatorTypeBox);
+    layout->addRow(new QLabel(tr("Number of interpolations"),widget),
+                   d->interpolationsBox);
+    layout->addRow(new QLabel(tr("Rho start"),widget),d->rhoStartBox);
+    layout->addRow(new QLabel(tr("Rho end"),widget),d->rhoEndBox);
+    layout->addRow(new QLabel(tr("Scaling coefficient"),widget),
+                   d->scalingCoeffBox);
+
+    // progression stack
+    d->progression_stack = new medProgressionStack(widget);
+
+    this->addWidget(widget);
+    this->addWidget(runButton);
+    this->addWidget(d->progression_stack);
+
+    //enable about plugin. Constructor called after the plugin has been registered, go ahead call it.
+    medPluginManager* pm = medPluginManager::instance();
```

Comment:
these 2 lines seem wrong. This is already called in `itkProcessRegistrationOptimusToolBox::plugin()` with an other name (Optimus), and these lines are not used here in other toolboxes